app/app/src/main/java/tech/ziasvannes/safechat/presentation/screens/profile/ProfileScreen.kt

```
@@ -198,26 +204,40 @@ fun ProfileScreen(onNavigateBack: () -> Unit, viewModel: ProfileViewModel = hilt
+                                                                val decodedBytes =
+                                                                        Base64.decode(
+                                                                                state.avatar,
+                                                                                Base64.DEFAULT
+                                                                        )
+                                                                BitmapFactory.decodeByteArray(
+                                                                                decodedBytes,
+                                                                                0,
+                                                                                decodedBytes.size
+                                                                        )
+                                                                        ?.asImageBitmap()
+                                                        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)